Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3248,6 +3248,9 @@ void WebPageProxy::processNextQueuedMouseEvent()
     if (!hasRunningProcess())
         return;
 
+    if (!m_mainFrame)
+        return;
```

Comment:
I'm observing a problem for Windows port.
If I move a mouse on a window immediately after MiniBrowser is started, all mouse events are ignored.
`WebPageProxy::processNextQueuedMouseEvent()` returns early because !m_mainFrame, then `WebPageProxy::processNextQueuedMouseEvent()` never be called.

Reply:
I confirmed this patch fixes the problem. 
~~~diff
diff --git a/Source/WebKit/UIProcess/WebPageProxy.cpp b/Source/WebKit/UIProcess/WebPageProxy.cpp
index e67f06539ee6..df4aefb34179 100644
--- a/Source/WebKit/UIProcess/WebPageProxy.cpp
+++ b/Source/WebKit/UIProcess/WebPageProxy.cpp
@@ -3212,6 +3212,9 @@ void WebPageProxy::handleMouseEvent(const NativeWebMouseEvent& event)
     if (!hasRunningProcess())
         return;
 
+    if (!m_mainFrame)
+        return;
+
 #if ENABLE(CONTEXT_MENU_EVENT)
     if (event.button() == WebMouseEventButton::RightButton && event.type() == WebEventType::MouseDown) {
         ASSERT(m_contextMenuPreventionState != EventPreventionState::Waiting);
~~~